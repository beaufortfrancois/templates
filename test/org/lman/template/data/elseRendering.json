{
  "foo": false,
  "bar": true,
  "qux": "qux\nxyzzy",
  "quux": false,
  "corge": false,
  "grault": true,
  "garply": false,
  "waldo": false
}