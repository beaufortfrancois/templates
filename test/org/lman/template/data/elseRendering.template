Before
{{?foo}}
foo is true
{{:}}
foo is false
{{/}}
{{^bar}}
bar is false
{{:bar}}
bar is true
{{/bar}}
baz is {{?baz}}{{baz}}{{:}}a tree{{/}}
qux is
{{?qux}}{{qux}}
{{:qux}}underwater
{{/qux}}
quux is
{{?quux}}
out in space{{:quux}}underwater
{{/quux}}
corge is
{{?corge}}
a dog{{:}}a cat{{/}}
grault is {{?grault}}true{{:}}{{/}}
garply is {{^garply}}{{:}}true{{/}}
waldo is {{?waldo}}{{:}}false{{/}}

    Indented
    {{?foo}}
    foo is true
    {{:}}
    foo is false
    {{/}}
    {{^bar}}
    bar is false
    {{:bar}}
    bar is true
    {{/bar}}
    baz is {{?baz}}{{baz}}{{:}}a tree{{/}}
    qux is
    {{?qux}}{{qux}}
    {{:qux}}underwater
    {{/qux}}
    quux is
    {{?quux}}
    out in space{{:quux}}underwater
    {{/quux}}
    corge is
    {{?corge}}
    a dog{{:}}a cat{{/}}
    After
