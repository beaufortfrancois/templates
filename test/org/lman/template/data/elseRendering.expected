Before
foo is false
bar is true
baz is a tree
qux is
qux
xyzzy
quux is
underwater
corge is
a cat
grault is true
garply is 
waldo is false

    Indented
    foo is false
    bar is true
    baz is a tree
    qux is
qux
xyzzy
    quux is
underwater
    corge is
a cat
    After
